src/rdposix.h

```diff
@@ -30,7 +29,8 @@
 /**
  * POSIX system support
  */
-#pragma once
+#ifndef RDPOSIX_H
```

Comment:
_<uppercase-headerfile_with_underscore_instead_of_dot>
that rule would yield
_RDPOSIX_H
is that what you meant or do you want
_RD_POSIX_H ?